php-nginx/entrypoint.sh

```diff
@@ -138,4 +134,7 @@ chmod -R 550 ${DOCUMENT_ROOT}
 # Change the www-data's shell back to /usr/sbin/nologin
 chsh -s /usr/sbin/nologin www-data
 
+# Remove the loose php-cli.ini
+rm ${PHP_DIR}/lib/php-cli.ini
```

Comment:
You can enable `exec` with `runtime:php`. This means it should be still as secure as possible even when you enable `exec`.

If it's for easier debugging, I can provide an alternate easy way later without loosen the security of the runtime.